libs/components/router/src/lib/modules/href/fixtures/href-fixture.module.ts

```diff
@@ -0,0 +1,19 @@
+import { CommonModule } from '@angular/common';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { NgModule } from '@angular/core';
+import { RouterTestingModule } from '@angular/router/testing';
+
+import { SkyHrefModule } from '../href.module';
+
+import { HrefDirectiveFixtureComponent } from './href-fixture.component';
+
+@NgModule({
+  declarations: [HrefDirectiveFixtureComponent],
```

Comment:
This is a prime candidate for a standalone component instead of a module.

Reply:
Yes, indeed. Happy to use the new stuff.